Source/DafnyCore/Backends/Python/PythonBackend.cs

```diff
@@ -100,6 +122,12 @@ public class PythonBackend : ExecutableBackend {
         return (false, null);
       }
     }
+
+    if (PythonModuleMode) {
+      Reporter.Info(MessageSource.Compiler, Token.Cli, "python build/run skipped in Python Module Mode");
+      return (true, null);
```

Comment:
Does Python also have issues with build / run in module mode? (Go did this because compiler doesn't generate the `go.mod` required to build / run)

Reply:
Let me take another look at this -- I copied this over and didn't think much about it